router/dealer.go

```diff
@@ -692,43 +701,47 @@ func (d *dealer) cancel(caller *Session, msg *wamp.Cancel) {
 
 	// Cancel mode should be one of: "skip", "kill", "killnowait"
 	mode := wamp.OptionString(msg.Options, "mode")
```

Comment:
It is not really clear what the default should be, since the meaning/workflow of each mode is not specified in the spec.  Also, looks like autobahn/crossbar uses "abort" instead of "killnowait".
https://github.com/guaracheitor/AutobahnPython/blob/master/autobahn/wamp/message.py#L1790

Cannot find any documentation of options, so was intentionally not rejecting unrecognized options, and defaulting to safe behavior of skip, since that cancels at the router, but does not send an INTERRUPT message to the callee.

So, what is preferred behavior?
1) Return error to caller on CANCEL with unrecognized mode
2) Log unrecognized mode and default unrecognized mode to "skip"